wire-ios/Configuration/url.json

```diff
@@ -1,5 +1,44 @@
 {
-    "wireAppOnItunes": "https://geo.itunes.apple.com/us/app/wire/id930944768?mt=8",
+    "_comment": {
+        "appOnItunes": "Link to the app on the App store.",
+        "support": "Link to the main support page.",
+        "searchSupport": "Link to the help desk support page.",
+        "website": "Link to the app's homepage.",
+        "emailAlreadyInUse": "Shown when the user tries to create an account with an email that is already in use. Links to support page explaining issue and what can be done.",
+        "whyToVerifyFingerprintArticle": "Link to an article explaining why a user should verify conversations using fingerprints.",
+        "howToVerifyFingerprintArticle": "Link to an article explaining how to manually verify conversations using fingerprints.",
+        "privacyPolicy": "Link to the privacy policy page.",
+        "legal": "Link to app's homepage for legal information, e.g terms of use and data processing addendum.",
+        "licenseInformation": "Link to the license information page.",
+        "passwordReset": "Link to the password reset page.",
+        "askSupportArticle": "Link to the support page where the user can submit a support request for various issues.",
+        "reportAbuse": "Link to the support page where a user can report an abuse issue.",
+        "wireEnterpriseInfo": "Link to an article explaining the various features of Wire.",
+        "legalHoldInfo": "Link to an article explaining legal hold.",
+        "guestLinksInfo": "Link to an article explaining how to create, share, and revoke a group conversation link.",
+        "unreachableBackendInfo": "Shown when a user tries to send a message or create a group with users from different backends when one of the backends is not reachable. Links to a support page that explains the issue when one of the backend is offline.",
+        "federationInfo": "Links to a support page explaining what Federation is.",
```

Comment:
so this is to comment what the links are. What about moving this to docC like with a markdown file instead.

Here, I found it confusing as it's not going to be parsed.

Reply:
@netbe and @johnxnguyen updated in the last commit